x-pack/plugins/maps/public/elasticsearch_geo_utils.js

```diff
@@ -451,3 +452,40 @@ export function clamp(val, min, max) {
     return val;
   }
 }
+
+export function extractFeaturesFromFilters(filters) {
+  const features = [];
+  filters
+    .filter(filter => {
```

Comment:
I'm wondering if there's another way to determine whether a filter is "spatial". Right now, manually added filters do _not_ show.

Reply:
as discussed, keeping this for a later improvement. Unless users edit DSL manually, ther;es no way in Kibana to add a spatial filter except for using Maps